apps/hip/static/js/sidebar.js

```diff
@@ -1,19 +1,38 @@
 export default function () {
-  const sideNavLinks = document.querySelectorAll(".sidenav-link-hip");
+  manageSideBarLinks();
+  closeButtonClicked();
```

Comment:
nitpick: I think I'd like this to be called `manageCloseButton` to match `manageSideBarLinks` (but you don't need to change it)

Reply:
done